lookout/style/format/features.py

```diff
@@ -435,24 +435,36 @@ def _inplace_write_vnode_features(
                         break
                     current_ancestor = parents[current_ancestor_id]
 
-            col_offset = 0
+            start_left = i - 1
```

Comment:
Would be nice to have a comment which explains this dark `+-=1` magic :)

Reply:
done